TOMSMorphingLabelExample/TOMSMorphingLabelExample/Base.lproj/Main.storyboard

```diff
@@ -1,9 +1,10 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
The storyboard wasn't changed manually. This just takes suggested changes from being run in Xcode 13, and bumps the storyboard version to support an iOS min of 7 to fix this error:
<img width="256" alt="Screen Shot 2021-08-16 at 3 30 54 PM" src="https://user-images.githubusercontent.com/10556242/129638972-4df92a96-1995-4097-835a-2359f06c1275.png">